dmtestmilstones.tex

```diff
@@ -44,18 +44,18 @@
 LDM-503-12 & 2020-01-31 & NCSA & \textbf{Verification tests in advance of pre-ops rehearsal for commissioning \#3}
 Dress rehearsal: commissioning starts in April, so by this stage we should be ready to do everything needed.
 \\ \hline
-LDM-503-13 & 2020-11-30 & NCSA & \textbf{Ops rehearsal DRP (ComCam data)}
+LDM-503-13 & 2020-11-30 & NCSA & \textbf{Verification test in advance of Ops rehearsal DRP (ComCam data)}
 ComCam data will now be available. Demonstrate its use in producing a data release.
 \\ \hline
 LDM-503-14 & 2021-03-31 & NCSA & \textbf{DM Software for Science Verification}
 Science Verification starts in April. Demonstrate that all required DM software is available.
 \\ \hline
-LDM-503-15 & 2021-11-30 & NCSA & \textbf{Ops rehearsal DRP (SV data)}
-Science Verification data will now be available. Demonstate its use in producting a data release.
+LDM-503-15 & 2021-11-29 & NCSA & \textbf{Verification tests in advance of full scale ops rehearsal }
+Science Verification data will now be available. Demonstrate its use in producing a data release.
 \\ \hline
-LDM-503-16 & 2022-02-28 & NCSA & \textbf{Verification tests in advance of full scale ops rehearsal \#1}
+LDM-503-16 & 2022-02-28 & NCSA & \textbf{Verification test in advance od Ops rehearsal DRP }
```

Comment:
I'd probably tweak the wording to "Verification test in advance of full DRP ops rehearsal".